WcaOnRails/config/locales/en.yml

```diff
@@ -1802,8 +1802,7 @@ en:
       it is highly recommended to search for sponsors. Although WCA Delegates are volunteers and do not receive a salary for their work, organizers should consider paying compensation for their travel expenses."
     paragraph3:
       title: "Deadlines for the announcement of the competition"
-      content_html: "The competition should be announced at least one month before the start of the competition (see <a href='https://www.worldcubeassociation.org/regulations/guidelines.html#8a4++'>Guideline 8a4++</a>). If there are special reasons preventing this deadline to be met, competitions may be announced up to two weeks before, at the discretion
-      of the WCAT. It is highly recommended to inform the WCAT of these reasons as soon as possible to avoid a later refusal of the competition."
+      content_html: "The competition must be announced at least 28 days before the start of the competition (see <a href='https://www.worldcubeassociation.org/documents/policies/external/Competition%20Requirements.pdf'>Competition Requirements</a>).
```

Comment:
I think you removed the closing quotation mark here, and that made this invalid yaml, see travis failure here https://travis-ci.org/thewca/worldcubeassociation.org/builds/567571944#L1451:

> /home/travis/.bundle/ruby/2.5.0/gems/i18n-1.6.0/lib/i18n/backend/base.rb:244:in `rescue in load_yml': can not load translations from /home/travis/build/thewca/worldcubeassociation.org/WcaOnRails/config/locales/en.yml: #<Psych::SyntaxError: (/home/travis/build/thewca/worldcubeassociation.org/WcaOnRails/config/locales/en.yml): did not find expected key while parsing a block mapping at line 1804 column 7> (I18n::InvalidLocaleData)